cypress/support/page_objects/navigationPage.js

```diff
@@ -0,0 +1,31 @@
+export class NavigationPage{
+
+    dropdownPage() {
+        cy.get('[data-test="sort-select"]').then( dropdown => {
+            cy.wrap(dropdown).click()
+            cy.get('[data-test="sort-option"]').each( (listItem, index) => {
+                const itemText = listItem.text().trim()
+
+                const titles = {
+                    "Brak": "Quest Cage Escape Room",
+                    "A-Z": "BLOKatowice",
+                    "Z-A": "Żurownia",
+                    "Ocena od najwyższej": "Quest Cage Escape Room",
+                    "Ocena od najniższej": "Komenda Wojewódzka Policji w Katowicach"
+                }
+
+                cy.wrap(listItem).click()
+                cy.wrap(dropdown).should('contain', itemText)
```

Comment:
I'm not sure if I understand PO Pattern well. Page objects should provide just objects used in tests and they shouldn't keep any assertions? Or maybe I'm wrong? :)

Reply:
In the examples that I saw in the test itself, there is only a method call, while the entire test management process with assertion is in the Page Object Patter, but I can be wrong ;-)